code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -215,7 +215,7 @@
 		else if(check_zone(M.zone_selected) == "head")
 			M.visible_message("<span class='notice'>[M] gives [src] a pat on the head to make [p_them()] feel better!</span>", \
 						"<span class='notice'>You give [src] a pat on the head to make [p_them()] feel better!</span>")
-			if(dna && dna.species && ((dna.features["tail_lizard"] != "None") || (dna.features["tail_human"] != "None") || ("mam_tail" in dna.species.mutant_bodyparts)))
+			if(dna && dna.species && ((src.dna.features["tail_lizard"] != "None") && (src.dna.features["tail_human"] != "None") && (src.dna.features["mam_tail"] != "None")))
```

Comment:
since, atm, the code is as good as commented out, it doesn't read any bodypart for the *wag